src/main/java/io/strimzi/kafka/bridge/http/HttpBridge.java

```diff
@@ -131,23 +131,14 @@ public void stop(Future<Void> stopFuture) throws Exception {
 
         this.isReady = false;
 
-        //Consumers cleanup
-        this.httpBridgeContext.getHttpSinkEndpoints().forEach((consumerId, httpEndpoint) -> {
-            if (httpEndpoint != null) {
-                httpEndpoint.close();
-            }
-        });
+        // Consumers cleanup
+        this.httpBridgeContext.closeAllSinkBridgeEndpoints();
         this.httpBridgeContext.getHttpSinkEndpoints().clear();
```

Comment:
The clear should be part of `closeAllSinkBridgeEndpoints`, since it doesn't make sense to leave endpoints in the map which have been closed. Likewise for the source endpoints below.